test/build.info

```diff
@@ -411,6 +412,20 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[dtlstest]=../include ../apps/include
   DEPEND[dtlstest]=../libcrypto ../libssl libtestutil.a
 
+  IF[{- !$disabled{sock} -}]
+    PROGRAMS{noinst}=bio_write_test bio_read_test
+  ENDIF
```

Comment:
This change is entirely redundant and can be removed. If `sock` is disabled then `dgram` automatically is too. Since you already have a check for `dgram` below - that is sufficient.